assets/constants.js

```diff
@@ -18,5 +18,214 @@ var constants = {
     "Statistics", "Trigonometry", "Welcome"],
   "DISABLED_EXPLORATION_IDS": ["5"],
```

Comment:
Done.
CAN_SEND_ANALYTICS_EVENTS moved to the top, the others were restored as suggested below.